### DISCUSSION TITLE
Random Quotes List

### DISCUSSION THREAD

**Discussion by AbdallahAbdedaiem** [lukePeavey/quotable#81] [Ideas]
I'm new to the api. So I'm not sure if this can be classified as an idea, not a question. Does the api support fetching random quotes list?

**Comment by lukePeavey** [comment]
I moved this to the ideas category. The API does not currently have a way to get a list of random quotes.   But I think this is a cool idea and maybe something we could add.

**Comment by raxraj** [comment]
Like a number of quotes at random?? Sounds like a good idea to me.
